counsel.el

```diff
@@ -3254,6 +3254,36 @@ Note: Duplicate elements of `kill-ring' are always deleted."
  '(("d" counsel-yank-pop-action-remove "delete")
    ("r" counsel-yank-pop-action-rotate "rotate")))
 
+;;** `counsel-evil-registers'
+
+(defcustom counsel-evil-registers-height 5
+  "The `ivy-height' of `counsel-evil-registers'."
+  :group 'ivy
+  :type 'integer)
+
+(defun counsel-evil-registers ()
+  "Ivy replacement for `evil-show-registers'."
+  (interactive)
+  (if (fboundp 'evil-register-list)
```

Comment:
Just a tip, not requesting it - you can save on indentation with
```el
(unless (fboundp 'evil-register-list)
  (user-error "..."))
```
as execution does not continue past errors.

Reply:
I think I need it so the byte compiler doesn't complain once I actually use that function.

Reply:
Again, not suggesting you actually do this here, but in such cases you can just [add a `declare-function` declaration](https://www.gnu.org/software/emacs/manual/html_node/elisp/Warning-Tips.html).

Reply:
Thanks, sounds good.